specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/examples/PostDeployWorkflowArtifacts.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "name": "testsite2",
+    "workflowName": "stateful2",
+    "slot": "staging",
+    "api-version": "2021-01-01"
+  },
+  "request": {
+    "body": {
+      "files": {
+        "stateful1/workflow.json": {
+          "definition": {
+            "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+            "actions": {},
+            "contentVersion": "1.0.0.0",
+            "outputs": {},
+            "parameters": {},
+            "triggers": {}
+          }
+        },
+        "connections.json": {
+          "managedApiConnections": {
+            "office365": {
+              "api": {
+                "id": "string"
+              },
+              "connection": {
+                "id": "string"
+              },
+              "connectionRuntimeUrl": "string",
+              "authentication": {
+                "type": "Raw",
+                "scheme": "Key",
+                "parameter": "@appsetting('office365-connectionKey')"
+              }
+            }
+          }
+        }
+      },
+      "appSettings": {
+        "SettingA": "A",
+        "SettingB": "B"
+      }
```

Comment:
Are these same settings accessible via any other API?

Reply:
Yep, there is a /config/appsettings endpoint. We update them in one request since the code change and the appsettings update both restart the site. This lets us get everything done in one restart rather than two.